### ISSUE TITLE
White text on light colored background in GNOME desiginated GTK3 apps. (Cinnamon Desktop)

### ISSUE THREAD

**Issue by superdreamkilla** [luigifab/human-theme#9]
GNOME styled apps have their own GTK3 setup that doesn't respect Cinnamon's default theme. We all know this, but there's another error I noticed.  Certain apps; so far only dconf Editor , have white text on a beige background that makes it hard to read.  This happens even if the Window is above all others. I really like old school themes, so I hope this bug is fixed.
![image](https://user-images.githubusercontent.com/61440491/104140033-35ab6180-53a7-11eb-8544-773ec77ec486.png)

**Comment by luigifab** [comment]
Here is a quick fix, you can add it in your _human-theme/gtk-3.0/user.css_ file.

```css
label.property-name-label,
label.key-name,
label.key-summary {
	color: @theme_text;
}

row:selected label.property-name-label,
row:selected label.key-name,
row:selected label.key-summary {
	color: @theme_selected_fg;
}
```

**Comment by luigifab** [comment]
Fixed in v1.2.0.
